docroot/profiles/custom/cgov_site/themes/custom/ncids_trans/templates/content/blog/node--cgov-blog-post--lb-primary-content.html.twig

```diff
@@ -30,7 +30,7 @@
     {{ drupal_block('cgov_blog_pager', {label: ''}) }}
     {% if content.field_recommended_content[0] %}
       <div id="blog-cards">
-        <h2 class="recommended-content__header">{{ series.field_recommended_content_header.value }}</h2>
+        <h4 class="recommended-content__heading">{{ series.field_recommended_content_header.value }}</h4>
```

Comment:
same here - you are reverting the change from https://github.com/NCIOCPL/cgov-digital-platform/pull/4415/files#diff-268249c03327c7a292d3fa33c13c15dab0cc9208c8378a82ff4ffbbf7507749eR33